src/lib.rs

```diff
@@ -871,12 +923,13 @@ impl Decimal {
 	}
 
 	pub fn log2(&self) -> f64 {
-		3.32192809488736234787 * self.log10()
+		LOG2_10
```

Comment:
`LOG2_10` is a constant. `Decimal::log2` should return the logarithm of base 2 of the current value. You probably meant to multiply it by `self.log10()`.

Reply:
Actually no ^^ Let me explain the change to that line:

The floating point precision of 3.32192809488736234787 is too large to fit in an `f64`, which is truncated down to 3.321928094887362 which in turn, when multiplied with `log10`, produces a very close approximation to the `LOG2_10` constant ^^ [It's explained here](https://rust-lang.github.io/rust-clippy/master/index.html#approx_constant)

`LOG2_10` is the value of `log2(10)`.

Reply:
Scratch my previous reply, I think I see what you mean.

Reply:
Yea, you're right, clippy was too overzealous here. Gonna amend that

Reply:
Fixed in dfa2f68